finch/processes/wps_xsubsetpoint_bccaqv2.py

```diff
@@ -0,0 +1,173 @@
+from pathlib import Path
+from pywps.response.execute import ExecuteResponse
+from pywps.app.exceptions import ProcessError
+from pywps.app import WPSRequest
+from .wpsio import start_date, end_date
+from pywps import LiteralInput, ComplexOutput, FORMATS, configuration
+
+from finch.processes import SubsetGridPointProcess
+from finch.processes.subset import SubsetProcess
+from finch.processes.utils import get_bccaqv2_inputs, netcdf_to_csv, zip_files
+
+
+class SubsetGridPointBCCAQV2Process(SubsetGridPointProcess):
+    """Subset a NetCDF file grid cells using a list of coordinates."""
+
+    def __init__(self):
+        inputs = [
+            LiteralInput(
+                "variable",
```

Comment:
I suggest putting inputs that are being reused in multiple processes in wpsio.py.

Reply:
I agree, but I will keep it as is until I remove the lat0 and lon0 for the subset_point process (we have to support both to update the process properly)

Reply:
ok